### ISSUE TITLE
Steaem overlay on every panel

### ISSUE THREAD

**Issue by TheGetHuber** [godotengine/godot#74772]
### Godot version

4.0dev

### System information

Windows 10, AMD FX-6200, 8 Gb RAM, GTX 1060

### Issue description

I have downloaded Steam version of Godot (4.0dev) and after i launch the project I see FPS counter on every panel (hints panel, main window, settings and ect.)

### Steps to reproduce

1) Download Steam version of Godot 4.0dev
2) Open the project
3) Done

### Minimal reproduction project

Not required. Works on any project.

**Comment by TheGetHuber** [comment]
Image of this bug:
![изображение](https://user-images.githubusercontent.com/69822538/224492522-62af7084-c8f4-448a-970d-093a2274e9bd.png)

**Comment by AThousandShips** [comment]
Are you unable to turn off the FPS counter? It should be a setting in Steam

**Comment by TheGetHuber** [comment]
Yes, i can disable it but bug's still here. I dont think that anybody will turn off this only for one thing.

**Comment by clayjohn** [comment]
We can probably forcibly disabled it in the editor like we do with MangoHud https://github.com/godotengine/godot/pull/71515

**Comment by Calinou** [comment]
[Check your Windows registry](https://renderdoc.org/vulkan-layer-guide.html) to see if there's a Steam overlay Vulkan layer JSON file. This should uncover an environment variable we can specify to disable the Steam overlay.

**Comment by HyperGamesDev** [comment]
Steam does force the overlay to be off, but weirdly enough the FPS counter is still there
![image](https://github.com/godotengine/godot/assets/48921794/dcbbdea8-bc5c-4ae6-a292-7783d15a436d)
![image](https://github.com/godotengine/godot/assets/48921794/5c653acb-5ac6-457d-8865-b770bf2499ab)

**Comment by BorisKlco** [comment]
Only workaround/fix I managed to do is... ☝🤓

While Godot not running, in Steam Librarary, click option on godot, General, disable Overlay while in-game.
Run Godot.
Exit Godot.

In Steam, Top-Left, Steam --> Settings --> In Game --> Disable everything in THE STEAM OVERLAY(6 toggles ☝🤓)
Run Godot. 
Exit Godot.

Again, Steam Settings --> In Game --> Enable Steam Overlay while in-game
in Steam Library, right click Godot --> Properties... --> Disable Steam Overlay while in-game
Run Godot.

I dont have any type of steam overlay, notification anymore in any Godot window, game debug, or option. ☝🤓

**Comment by Calinou** [comment]
> Steam does force the overlay to be off, but weirdly enough the FPS counter is still there ![image](https://user-images.githubusercontent.com/48921794/268614120-dcbbdea8-bc5c-4ae6-a292-7783d15a436d.png) ![image](https://user-images.githubusercontent.com/48921794/268614240-5c653acb-5ac6-457d-8865-b770bf2499ab.png)

I believe the FPS counter is displayed by a separate option in the Steam settings – it's not part of the Steam overlay.

**Comment by YuriSizov** [comment]
This can still happen as of 4.2. Interesting that opening a window causes the counter to scale dramatically (and closing it restores it back to a normal size). This might indicate an issue with viewport scaling, incidentally.
